examples/go-client/api-admin.yaml

```diff
@@ -1,40 +0,0 @@
-kind: user
```

Comment:
Same with these example roles, they're not needed by documentation anymore?

Reply:
This fille is referenced in the docs, so I'll change that to an inline.